themes/demuxed-2022/template.html

```diff
@@ -228,12 +228,16 @@
 </style>
 
 <media-controller
+  breakpoints="xs:360 sm:600 md:760 lg:960 xl:1100"
+  nodefaultstore="{{nodefaultstore}}"
   defaultsubtitles="{{defaultsubtitles}}"
+  defaultduration="{{defaultduration}}"
   gesturesdisabled="{{disabled}}"
   hotkeys="{{hotkeys}}"
   nohotkeys="{{nohotkeys}}"
   audio="{{audio}}"
-  breakpoints="xs:360 sm:600 md:760 lg:960 xl:1100"
+  defaultstreamtype="on-demand"
+  exportparts="layer, media-layer, poster-layer, vertical-layer, centered-layer, gesture-layer"
```

Comment:
I feel a little hesitant to export parts because of the implications on versioning these themes. Is there a specific driver for this?